### PR TITLE
Add `pairing_allowed` column to Section, defaults to true

### PR DESCRIPTION
**Pull request by joshlory**
Spec: https://docs.google.com/document/d/1jOHX8-wacgpbRy9bjnwZi1zlSJJ9MZ5RpejFZyWMv08/edit

### CODE REVIEW

File: dashboard/db/migrate/20170524215951_add_pair_programming_enabled_to_section.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddPairProgrammingEnabledToSection < ActiveRecord::Migration[5.0]
+  def change
+    add_column :sections, :pair_programming_enabled, :boolean, null: false, default: true
```
Comment (ashercodeorg) [inline dashboard/db/migrate/20170524215951_add_pair_programming_enabled_to_section.rb]: Is `enabled` redundant? More succinct and equally descriptive (at least to me) would be `pairing_allowed` or `pair_programming`.

**Comment by joshlory** [comment]
Good catch, fixed.  PTAL.

**Comment by ashercodeorg** [comment]
LGTM. Is this PR a candidate for an off-hours migration? If so, life is easier (from experience) if you add the changed `schema_cache.dump` file  to this PR. Useful rake commands are `rake db:schema:cache:clear` and `rake db:schema:cache:dump`.

**Comment by ashercodeorg** [comment]
This PR is being closed in favor of #15456, which merges staging and fixes the timestamp conflicts.

**Pull request closed without merging**